### ISSUE TITLE
Use of ES2016 exponent operator (**) in dist/d3-geo.js

### ISSUE THREAD

**Issue by melnikov-s** [d3/d3-geo#166]
This is breaking older browsers as we do not run babel against node_modules. We can make an exception for d3 but since we never had to before and the package just got released I'm assuming this is a mistake? 

The use of the operator originates from `container.js`

**Comment by Gary-Li** [comment]
Hello, same problem is happening to me also. 
```
ERROR in commons.js from UglifyJs

Unexpected token: operator (*) [./node_modules/d3-geo/src/contains.js:72,0][commons.js:225669,48]

```

**Comment by scinconovetta** [comment]
Getting this error when building new Kibana-plugin packages this afternoon (note: this was not happening this morning):
```
 ERROR in /usr/share/kibana/plugins/<plugin name>/~/d3-geo/dist/d3-geo.js
    Module parse failed: /usr/share/kibana/plugins/<plugin name>/node_modules/d3-geo/dist/d3-geo.js Line 1693: Unexpected token *
    You may need an appropriate loader to handle this file type.
    |         ao <= ab &&
    |         bo <= ab &&
    |         (ao + bo - ab) * (1 - ((ao - bo) / ab) ** 2) < epsilon2 * ab
    |       )
    |         return true;
     @ /usr/share/kibana/plugins/<plugin name>/~/react-vis/dist/plot/series/contour-series.js 21:13-30
```

Version of d3-geo is 1.11.5

**Comment by scinconovetta** [comment]
@Fil can you take a look at this? This is used as a dependency of a dependency of a ... you get the point ... and I can't work around it.

**Comment by mbostock** [comment]
This is a bug. We plan to adopt ES2015 syntax in the 2.0 release, but this should not have been added to 1.x.

**Comment by scinconovetta** [comment]
I worked around this by adding `d3-geo@1.11.3` as a dependency to my project's `package.json`

**Comment by mbostock** [comment]
Fixed in 1.11.6.

**Comment by Fil** [comment]
Sorry! Lesson learned for next time 😰
